system/mount.py

```diff
@@ -358,13 +358,17 @@ def main():
                     res,msg = mount(module, **args)
             elif 'bind' in args.get('opts', []):
                 changed = True
-                cmd = 'mount -l'
+                cmd = 'findmnt -rn %s' % args['name']
```

Comment:
the original code and this should use find_bin function, which would allow for fallback in the case findmnt does not exist.

i would still prefer a fix that works for all cases, but I guess we can include this for now